src/styles/blocks/section3.scss

```diff
@@ -0,0 +1,31 @@
+.section_3 {
+  display: flex;
+  height: 40px;
+  width: 166px;
+  justify-content: stretch;
+}
+
+.section_3__link {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 12px 68px;
+  height: 40px;
+  background-color: #00acdc;
+  flex-grow: 1;
+  text-align: center;
+  justify-content: center;
```

Comment:
The 'justify-content' property is not effective on a non-flex container. Since '.section_3__link' is not displayed as flex, this property can be removed.